src/ui/public/agg_types/param_types/filter/field_filters.ts

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { FieldParamType } from '../';
+import { AggType } from '../..';
+import { IndexPattern } from '../../../index_patterns';
+import { AggConfig } from '../../../vis';
+
+type AggTypeFieldFilter = (
+  fields: any[],
```

Comment:
This signature is wrong. It should be `field: any`. TypeScript don't catch this error, since we use `any` in this place, and `any[]` is always assignable to `any` anyways.

Reply:
fixed!